srcpkgs/66-boot/REMOVE

```diff
@@ -0,0 +1,6 @@
+case "${ACTION}" in
+	post)
+		66-tree -R boot
+		66-tree -R default
```

Comment:
I don't think this should be done in REMOVE, I assume this removes user data?

Reply:
Yes, it removes userdata. Removed.